test/test_torch.py

```diff
@@ -10270,6 +10276,88 @@ def test_exponential(self, device, dtype):
         with self.assertRaises(RuntimeError):
             torch.empty((1,), device=device, dtype=dtype).exponential_(-0.5)
 
+    @unittest.skipIf(not TEST_SCIPY, 'Scipy not found')
```

Comment:
Please consider adding `@skipIfNoSciPy` macro to `common_utils.py`

Reply:
Moved to #38299